codespell_lib/data/dictionary.txt

```diff
@@ -2032,7 +2033,7 @@ aquiring->acquiring
 aquisition->acquisition
 aquit->acquit
 aquitted->acquitted
-aquries->acquries
+aquries->acquires
```

Comment:
I wonder if this should suggest the phonetic equerries too?

Reply:
I'm not sure. But I'm not a native :-)

Reply:
I think acquires and equerries are phonetically different enough that you'd be **very** unlikely to misspell one as the other.

Reply:
Yes, although that's the wrong comparison @lurch . They've actually written `aquries`, which I'd suggest is possibly reasonably phonetically similar to `equerries`

Reply:
i added it